content/en/blog/2025-09-29-announcing-flux-v2.7.0/index.md

```diff
@@ -0,0 +1,383 @@
+when running Flux in the hub-and-spoke mode.
+
+This feature allows cluster admins to use cloud identities on the hub cluster to 
+configure Flux authentication to spoke clusters, without the need to create and manage
+static `kubeconfig` Secrets.
+
+For more details on how to configure secret-less authentication to remote clusters,
+please refer to the following guides:
+
+- [Kustomization - remote cluster apply](/flux/components/kustomize/kustomizations/#secret-less-authentication)
+- [HelmRelease - remote cluster apply](/flux/components/helm/helmreleases/#secret-less-authentication)
+
+## Object-level Workload Identity
+
+In Flux v2.7, we have completed the integration of Kubernetes Workload Identity
+at the object level for all Flux APIs that support authentication with cloud providers.
+
+This includes the following resources:
+
+- `Bucket.spec.serviceAccountName` for authenticating with AWS S3, Azure Blob Storage and Google Cloud Storage.
+- `GitRepository.spec.serviceAccountName` for authenticating with Azure DevOps.
+- `OCIRepository.spec.serviceAccountName` for authenticating with AWS ECR, Azure Container Registry and Google Artifact Registry.
+- `ImageRepository.spec.serviceAccountName` for authenticating with AWS ECR, Azure Container Registry and Google Artifact Registry.
+- `Kustomization.spec.decryption.serviceAccountName` for authenticating with AWS KMS, Azure Key Vault and Google KMS.
+- `Kustomization.spec.kubeConfig.configMapRef.name` for authenticating with remote clusters on AWS EKS, Azure AKS and Google GKE.
+- `HelmRelease.spec.kubeConfig.configMapRef.name` for authenticating with remote clusters on AWS EKS, Azure AKS and Google GKE.
+- `Provider.spec.serviceAccountName` for authenticating with Azure DevOps, Azure Event Hub and Google Pub/Sub.
+
+For more details on how to configure object-level workload identity for Flux, see the following docs:
+
+- [AWS workload identity](/flux/integrations/aws/)
+- [Azure workload identity](/flux/integrations/azure/)
+- [GCP workload identity](/flux/integrations/gcp/)
+
+## OpenTelemetry Tracing
+
+Starting with Flux v2.7, users can enable OpenTelemetry tracing for Flux reconciliations
+by configuring a Provider of type `otel`:
+
+```yaml
+apiVersion: notification.toolkit.fluxcd.io/v1beta3
+kind: Provider
+metadata:
+  name: jaeger
+  namespace: flux-system
+spec:
+  type: otel
+  address: http://jaeger-collector.jaeger:4318/v1/traces
+```
+
+The notification-controller converts Flux events into OTEL spans with proper trace relationships
+based on the Flux object hierarchy. Source objects (`GitRepository`, `HelmChart`, `OCIRepository`, `Bucket`)
+create root spans, while `Kustomization`and `HelmRelease` objects create child spans within the same trace.
+Each span includes event metadata as attributes and uses the alert name and namespace as the service identifier.
+
+![fluxcd-otel](flux-otel-trace.png)
```

Comment:
Everything is really tiny in this screenshot. It's nearly impossible to recognize anything.

Reply:
Right, you need to zoom in on your browser to see what's going on. There must be a Hugo plugin that shown images full screen...

Reply:
There is: https://github.com/fluxcd/website/blob/18fccdbcdd61bc19885f3717f7a08da1909b3675/content/en/blog/2023-08-28-announcing-flux-v2.1.0/index.md#flux-grafana-dashboards

This might help seeing better what's going on.